src/RichError.tsx

```diff
@@ -31,7 +31,11 @@ const OpenElsewhere: FC = () => {
   const { t } = useTranslation();
 
   return (
-    <ErrorView Icon={PopOutIcon} title={t("error.open_elsewhere")}>
+    <ErrorView
+      widget={null}
```

Comment:
It shouldn't normally be possible for a widget to encounter this error but, just to avoid surprises:

```suggestion
      widget={widget}
```